.github/workflows/ci.yaml

```diff
@@ -360,9 +360,9 @@ jobs:
 
       # Decide which Docker Meta job to run based on whether this scanner is a third party scanner, whose tag should be the Chart's appVersion
       - name: Docker Meta
-        id: docker_meta
+        id: docker_meta_scanner
+        if: hashFiles(format('./scanners/{0}/scanner/Dockerfile', matrix.unit)) != ''
         uses: crazy-max/ghaction-docker-meta@v1
-        if: contains(fromJson('["angularjs-csti-scanner", "gitleaks", "kube-hunter", "kubeaudit", "ncrack", "nmap"]'), matrix.unit)
```

Comment:
I believe this manual check was necessary. For example, the git-repo-scanner has its own Dockerfile but is always tagged with the secureCodeBox tag, and not the appVersion (which is 1.0 for git-repo-scanner).

In the old Pipeline, there was a distinction between third-party scanners and custom scanners. Both have a Dockerfile, but third-party scanners follow their own versioning, and custom scanners follow secureCodeBox versioning.